archive/100-theorems-list/42_inverse_triangle_sum.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2020. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jalex Stark, Yury Kudryashov
+-/
+import tactic
+import algebra.big_operators
+import data.real.basic
+
+/-!
+The problem is referred to as "Sum of the Reciprocals of the Triangular Numbers" on the 100 theorems list.
```

Comment:
```suggestion
# Sum of the Reciprocals of the Triangular Numbers

This file proves Theorem 42 from the [100 Theorems List](https://www.cs.ru.nl/~freek/100/).
```
The module doc at the start of a file should start with a title at heading level 1.